crates/orderbook/src/database/orders.rs

```diff
@@ -367,33 +364,12 @@ impl LimitOrderCounting for Postgres {
             .start_timer();
 
         let mut ex = self.pool.acquire().await?;
-        Ok(database::orders::user_orders_with_quote(
```

Comment:
@MartinquaXD the orders which are real "in-market" orders should be executed relatively soon (minutes I believe), leaving the database populated only with real `limit` orders very soon. If we don't do a migration, the risk is to hitting the too many limit orders because we may count "in-market" orders as limit orders, but the question is for how long will those orders live?
Otherwise, we would need to do a migration which goes through all the orders, fetch the quote, and check if they are "in-market" or not to reclassify them, which I can also do, I was just wondering if we would really need it. WDYT?

Reply:
While I think you are right that we'd not run into order counting issues for long it would still be good to retroactively label orders correctly so we have a consistent state in the DB.
Otherwise people will have to remember that the meaning of the label changed at some point when they look into some old order (which can happen if we do some historic analysis).

Reply:
oh, that makes a lot of sense! got it! perfect, I will prepare a migration for it then! thank you!